src/App.tsx

```diff
@@ -1,35 +1,35 @@
 import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
+import { SearchBar } from './components/SearchBar/SearchBar';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
-
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+    movies: moviesFromServer,
+  };
+
+  changeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      query: event.target.value,
+    });
+  };
 
   render() {
+    const { movies, query } = this.state;
+    const filteredMovies = movies
+      .filter(movie => (
+        (movie.title + movie.description).toLowerCase()
```

Comment:
Not really sure if concatenation of this string is a good idea, but it's subjective)

Reply:
Not the best solution, I do agree on that. It just made this line of code much shorter😅 
But if I understand the concepts right, the time complexity would be equal in both cases, since string concatenation takes constant time. But I should check on that on Q&A. Google keeps giving me different answers